build_projects/dotnet-cli-build/CompileTargets.cs

```diff
@@ -157,7 +158,22 @@ private static void RemovePdbsFromDir(string directory)
             FS.RmFilesInDirRecursive(directory, "*.pdb");
         }
 
-        private static BuildTargetResult CompileCliSdk(BuildTargetContext c, DotNetCli dotnet, string rootOutputDirectory)
+        private static BuildTargetResult CompileCliSdkAndGenerateNuGetPackagesArchive(
```

Comment:
Should we just rename the CompileCliSdk method to this?

This is only calling CompileCliSDK